homework/01LessonOne.js

```diff
@@ -203,220 +205,220 @@ var stringFourStrictNotEqualsNumberFour;
 // What’s the difference between == and ===?
 //
 // Your Answer Goes Here:
-//
+// ANSWER:  == is for weak equality (checks for loose equality); === is for strict equality.
 //
 // What’s the difference between != and !==?
 //
 // Your Answer Goes Here:
-//
+//  ANSWER:  != is not equal; !== is not equal value or not equal type
```

Comment:
Almost! It's a bit simpler than that
!= is the inverse comparison of == (aka not equal)
!== is the inverse comparision of === (you should think of this as not-strict-equal)